src/nxt_clone.c

```diff
@@ -387,8 +387,25 @@ nxt_clone_vldt_credential_gidmap(nxt_task_t *task,
     }
 
     base_ok = 0;
-    gids_ok = 0;
+    for (i = 0; i < map->size; i++) {
+        m = map->map[i];
 
+        if (creds->base_gid >= (nxt_gid_t) m.container
+            && creds->base_gid < (nxt_gid_t) (m.container+m.size))
```

Comment:
I know it was like this already, but the general style seems to be to have a space around numerical operators.

Reply:
Now I've force pushed to change the patch subject, to clarify that it only _moves_ code.  No changes; not even whitespace.

Reply:
Ah, OK.